lib/content-services/src/lib/permission-manager/components/permission-list/permission-list.component.html

```diff
@@ -73,6 +74,7 @@ <h3>{{'PERMISSION_MANAGER.LABELS.DIRECT-PERMISSIONS' | translate }}</h3>
                 <button
                     mat-button
                     [matTooltip]="'PERMISSION_MANAGER.ACTION.ADD-PERMISSION' | translate"
+                    title="{{'PERMISSION_MANAGER.ACTION.ADD-PERMISSION' | translate}}"
```

Comment:
this might be conflicting with the matTooltip above... probably remove one of those

Reply:
noted and resolved